src/main/scala/collatz/Collatz.scala

```diff
@@ -1 +1,12 @@
+/** Given positive integer n, computes collatz function */
+def collatz(n: Int) =
+    if n % 2 == 0 then n / 2
+    else (3 * n) + 1 
 
+/** Given positive integer n, computes number of calls to 
+ * collatz until n equals 1 
+ */
+def collatzCount(n: Int): Int =
+    if n == 1 then 0
+    else if collatz(n) == 1 then 1
+    else 1 + collatzCount(collatz(n))
```

Comment:
That's interesting that you included the collatz(n) == 1 like a potential base case - I did not explicitly account for that case since the else case would. On the other hand, your version is more readable than mine.